docs/topics/cli-run.adoc

```diff
@@ -52,7 +52,7 @@ The following command analyzes the `org.jboss.seam` packages of the link:https:/
 
 [source,terminal,subs="attributes+"]
 ----
-$ <{ProductShortName}_HOME>/bin/windup-cli --sourceMode --input /path/to/seam-booking-5.2/ \
+$ <{ProductShortName}_HOME>/bin/{mta-cli} analyze --sourceMode --input /path/to/seam-booking-5.2/ \
```

Comment:
The `sourceMode` argumen has been replaced by `mode`

    -m, --mode string               analysis mode. Must be one of 'full' or 'source-only' (default "full")

Reply:
My bad I didn't clarify, the example command would be 

    {mta-cli} analyze --mode source-only ...